python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -348,6 +361,14 @@ def global_avgpooling(attrs, inputs, proto_obj):
                                                                 'pool_type': 'avg'})
     return 'Pooling', new_attrs, inputs
 
+def global_lppooling(attrs, inputs, proto_obj):
+    """Performs global lp pooling on the input."""
+    p_value = attrs.get('p', 2)
+    new_attrs = translation_utils._add_extra_attributes(attrs, {'global_pool': True,
```

Comment:
the indentation is bad here, two suggested way from PEP8 is 
```
a = super_long_function_name(
    xxx, xxx, xxx, xxx, xx, xxx)

# or 
a = super_long_function_name(xxx, xxx, xxx,
                             xxx, xxx, xxx)

```
I suggest to use the first style here to avoid too many lines

Reply:
the one I have used is the second style and it has passed pylint and it is good code readability-wise. I would ideally want the dict to be printed as name, value pairs rather than as a single long list.